packages/data/src/world/mm/overworld.yml

```diff
@@ -1392,11 +1392,10 @@
   exits:
     "Swamp Back": "true"
     "Deku Palace Cliff": "has(MASK_DEKU) || short_hook_anywhere"
-    "Near Deku Shrine": "event(CLEAN_SWAMP)"
+    "Near Deku Shrine": "event(CLEAN_SWAMP) || can_use_nayru"
```

Comment:
should check for zora with nayru.

Reply:
Do you mean because zora is required for killing the big octo?